### ISSUE TITLE
Data Access

### ISSUE THREAD

**Issue by nmduonggg** [MSKCC-Computational-Pathology/DMMN-osteosarcoma#1]
Thanks for your contribution to pathology research. I am highly interested in your work and want to access the data for further pretrain and finetune. Is there a way to access your data including HE images and their labels ?

**Comment by nikhithajoy** [comment]
This is a fantastic project. Would it be possible to provide access to the dataset along with its metadata?
